Fixed the table azure_security_center_contact to populate the column value correctly Closes #900

ParthaI opened the pull request:
# Integration test logs
<details>
  <summary>Logs</summary>
  
```
N/A
```
</details>

# Example query results
<details>
  <summary>Results</summary>
  
```
> select * from azure_security_center_contact
+-----------------------------------------------------------------------------------------------------------+---------+-------------------------------------+---------------------+-----------------+------------+----------------------------------+---------------------------------------------------+---------+--------->
| id                                                                                                        | name    | type                                | email               | phone           | is_enabled | notifications_by_role            | notifications_sources                             | title   | akas    >
+-----------------------------------------------------------------------------------------------------------+---------+-------------------------------------+---------------------+-----------------+------------+----------------------------------+---------------------------------------------------+---------+--------->
| /subscriptions/************************************/providers/Microsoft.Security/securityContacts/default | default | Microsoft.Security/securityContacts | contact@example.com | +1-555-555-5555 | true       | {"roles":["Owner"],"state":"On"} | [{"minimalSeverity":"High","sourceType":"Alert"}] | default | ["azure:>
+-----------------------------------------------------------------------------------------------------------+---------+-------------------------------------+---------------------+-----------------+------------+----------------------------------+---------------------------------------------------+---------+--------
```
</details>

misraved commented:
@ParthaI please take a look at the following review comments:
- `alert_notifications` and `alerts_to_admins` should be deprecated first and not removed. Please add the deprecated message to the description of the columns.
- Did you add `github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/security/armsecurity v0.14.0` directly as a dependency?
- I see a bunch of tables using the security package - https://hub.steampipe.io/plugins/turbot/azure/tables?q=azure_security_center_, do they need updates as well?

ParthaI commented:
@misraved,

- I’ve added the deprecation notice to the column descriptions.
- Regarding the dependency: yes, I added it directly for the `azure_security_center_contact` table only. If I will upgrade the package `github.com/Azure/azure-sdk-for-go/profiles/preview/preview/security/mgmt/security`, it might be issue for the other tables. BTW Azure recommends using the new ARM package, we're currently in the process of transitioning all plugin code to adopt the ARM package gradually.
- I also ran queries against the other tables and confirmed that they are working correctly.

Please take another look at the PR and let me know if you have any concerns. Thanks!
